include_list.txt

```diff
@@ -0,0 +1,2 @@
+# This is a list of Python additional packages needed. This file is used with pip.
+netifaces
```

Comment:
If we add such item here then we need to comment why we did this. Otherwise, we don't know if or when can we removed if from the list. Is this not a dependency of pytest or ci packages?

I think Marek needed this just for one architecture.


This reminds me that I've proposed a different format for these files in IDF-8157. 

```
The format could be a YML file with records about the python version, architecture, comment and the package requirement (name and version) [also could be a list]. The python version and architecture could be "*" or a list of desired ones. Each build action then could check if it is in the list to build the package.
```
It is OK to do it later if you haven't forget about it.